src/modules/Dashboard/features/RespondentData/RespondentDataReview/ResponsesSummary/ResponsesSummary.hooks.ts

```diff
@@ -26,11 +30,16 @@ export const useResponsesSummary = ({
     },
     {
       id: 'review-desc-2',
+      title: t('respondentIdentifierWithColon'),
+      content: respondent,
+    },
+    {
```

Comment:
Adding another column here is causing the last column to wrap to the next row, but in Figma all columns should be on the same row:

| Figma | This PR | 
| - | - |
| ![CleanShot 2024-11-06 at 10 37 01@2x](https://github.com/user-attachments/assets/3b22480f-91ee-44d2-98cb-f27a28459542) | ![CleanShot 2024-11-06 at 10 36 54@2x](https://github.com/user-attachments/assets/edab8b75-380b-4ded-b914-58a2c2081aef) |

Might just need to update the `columns` prop in the `ResponsesSummary` component!

Also I noticed that the labels and the values have new styling in Figma – the labels have `title-bold/small` (whatever that equates to in our theme tokens) while the values have `body/large`.